src/openms/source/DATASTRUCTURES/String.cpp

```diff
@@ -526,5 +526,11 @@ namespace OpenMS
     this->append(s);
     return *this;
   }
+  
+  std::size_t hash_value(const OpenMS::String & s) noexcept
```

Comment:
let's see if someone chokes on that :)

Reply:
Well someone else already added it to the String.h. It is definitely necessary, otherwise you can't use boost::hash.